task/upload.go

```diff
@@ -244,6 +245,21 @@ func processCatalystCallback(tctx *TaskContext, callback *clients.CatalystCallba
 			})
 		case OutputNameIPFSSourceMP4:
 			assetSpec.Storage.IPFS.CID = output.Manifest
+		case OutputNameAssetMP4:
+			if len(output.Videos) != 1 {
+				return nil, fmt.Errorf("unexpected number of videos in source MP4 output: %d", len(output.Videos))
+			}
+			for _, video := range output.Videos {
+				if video.Type != "mp4" {
+					return nil, fmt.Errorf("unexpected video type in source MP4 output: %s", output.Videos[0].Type)
+				}
+				videoFilePath = video.Location
+				assetSpec.Files = append(assetSpec.Files, api.AssetFile{
+					Type: "static_file",
```

Comment:
WDYT of `"static_mp4` or even `static_transcoded_mp4` to be more specific? The other file we have is `catalyst_hls_manifest` in order to have a very clear meaning and processing logic. I think the more specific we are here the easier it will be to maintain without needing additional fields in this `asset.files` list, so no need for a generic name like "static file" IMO.

Additionally, we should add _some_ metadata about the file here. The MVP would be specifying the file size, but ideally we could have more video-specific info as well like resolution etc.

Reply:
I thought about it, I opted to left out `mp4` as in the future we may want to support other containers, specifically `webm` as, although is not widely supported, it's more modern and produce smaller files which could benefit the TTF & load time. However, since we are specifying `mp4` in other places, we can be specific also here and do something generic later and if needed, so I agree with this, changing it!

Reply:
I changed it as well on `studio` playback